gradle-tests/src/main/java/org/arquillian/example/Greeter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2012, 2018, IBM Corporation, Red Hat Middleware LLC, and individual contributors
+ * Copyright 2012, 2018, 2021, IBM Corporation, Red Hat Middleware LLC, and individual contributors
```

Comment:
When updating copyright years you are supposed to change the last year, not add another year.